sources/install/package_mobile.sh

```diff
@@ -90,6 +90,16 @@ function install_androguard() {
     add-to-list "androguard,https://github.com/androguard/androguard,Reverse engineering and analysis of Android applications"
 }
 
+function install_mobsf(){
+    colorecho "Installing Mobile Security Framework"
+    fapt wkhtmltopdf
+    git -C /opt/tools clone --depth 1 https://github.com/MobSF/Mobile-Security-Framework-MobSF
+    python3 -m pipx install /opt/tools/Mobile-Security-Framework-MobSF/
+    add-history mobsf
+    add-test-command "/root/.local/pipx/venvs/mobsf/bin/python -c 'from mobsf.MobSF.settings import VERSION; print(VERSION)'"
+    add-to-list "mobsf,https://github.com/MobSF/Mobile-Security-Framework-MobSF,Automated, all-in-one mobile application (Android/iOS/Windows) pen-testing, malware analysis and security assessment framework"
```

Comment:
Please remove all `,` in the tool description, this line can only have two `,`

Reply:
done